bears/scss/SCSSLintBear.py

```diff
@@ -21,4 +23,265 @@ class SCSSLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return filename,
+        return filename, '--config', config_file
+
+    @staticmethod
+    def generate_config(filename, file,
+                        check_bang_format: bool=True,
+                        space_before_bang: bool=True,
+                        space_after_bang: bool=False,
+                        allow_chained_classes: bool=False,
+                        allow_color_keywords: bool=False,
+                        use_color_variables: bool=True,
+                        allow_debug_statement: bool=False,
+                        check_declaration_order: bool=True,
+                        allow_duplicate_properties: bool=False,
+                        allow_consecutives_duplicate_property: bool=False,
+                        else_placement_style: str='same_line',
+                        force_empty_line_between_blocks: bool=True,
+                        allow_empty_rules: bool=False,
+                        hexadecimal_length_style: str='short',
+                        hexadecimal_notation_case: str='lowercase',
+                        allow_hexadecimal_validation: bool=True,
+                        allow_id_selector: bool=False,
+                        allow_important_rule_in_properties: bool=False,
+                        use_spaces: bool=True,
+                        indent_size: int=2,
+                        leading_zero: str='exclude_zero',
+                        allow_mergeable_selectors: bool=False,
+                        allow_leading_underscore: bool=True,
+                        function_naming_convention: str='hyphenated lowercase',
+                        mixin_naming_convention: str='hyphenated lowercase',
+                        variable_naming_convention: str='hyphenated lowercase',
+                        placeholder_naming_convention:
+                            str='hyphenated lowercase',
+                        max_depth: int=3,
+                        use_placeholder_selector_in_extend: bool=True,
+                        max_properties: int=10,
+                        allow_unit_on_zero_values: bool=False,
+                        check_ulrs_format: bool=True,
+                        urls_in_quotes: bool=True,
+                        allow_unnecesseary_parent_reference: bool=False,
+                        allow_unnecessary_mantissa: bool=False,
+                        allow_trailing_whitespaces: bool=False,
+                        allow_trailing_semicolon: bool=True,
+                        check_imports_path: bool=True,
+                        allow_filename_leading_underscore: bool=False,
+                        allow_extension: bool=False,
+                        use_length_variables: bool=True,
+                        check_properties_spelling: bool=True,
+                        extra_properties: list=[],
```

Comment:
We don't use mutables like lists for default arguments.
Hence this should be `list=(),` or list=None and later is None has been given, convert it to a []
https://github.com/coala-analyzer/coala/issues/452